### PR TITLE
Recommend users to configure ntp_servers

### PR DESCRIPTION
**Pull request by FrankYang0529**
Related issue: https://github.com/harvester/harvester/issues/2119

### CODE REVIEW

File: docs/install/iso-install.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
 1. Configure the `Virtual IP` which you can use to access the cluster or join other nodes to the cluster <small>(Note: If your IP address is configured via DHCP, you will need to configure static MAC-to-IP address mapping on your DHCP server in order to have a persistent Virtual IP)</small>.
 1. Configure the `cluster token`. This token will be used for adding other nodes to the cluster.
 1. Configure the login password of the host. The default SSH user is `rancher`.
-1. (Optional) Configure the NTP Servers of the node. This defaults to `0.suse.pool.ntp.org`.
+1. (Recommended) Configure the NTP Servers of the node. This defaults to `0.suse.pool.ntp.org`.
```
Comment (guangbochen) [inline docs/install/iso-install.md:36]:
```suggestion
1. Recommended configuring the NTP server to make sure all nodes' times are synchronized. This defaults to `0.suse.pool.ntp.org`
```

Comment (guangbochen) [reply docs/install/iso-install.md:36]: also need to update the same paragraph on the [index page](https://github.com/harvester/docs/blob/main/docs/index.md).

Comment (FrankYang0529) [reply docs/install/iso-install.md]: Fixed it. Thank you.

**Review by guangbochen — APPROVED** [review]

**Review by weihanglo — APPROVED** [review]
LGTM

**Pull request merged**